### ISSUE TITLE
Use dynamic JRE packages than packaged sugar JRE

### ISSUE THREAD

**Issue by srevinsaju** [163gal/Java#3]

**Comment by quozl** [comment]
More detail needed before I can comment.

**Comment by srevinsaju** [comment]
Regarding #2, you have provided the information that this activity was meant only for OLPC OSes, I actually meant these could be installed System wide rather than depending on a src hardcoded on the repository.

**Comment by quozl** [comment]
Thanks.  Yes, Java would be installed system-wide as first preference.

**Comment by srevinsaju** [comment]
@quozl , unrelated, how do we know if the 
(i) activity is maintained (or if work on activity has been stopped, with no intention to maintain in?
(ii) The activity is only for OLPC OS?
Thanks

**Comment by quozl** [comment]
Either ask, do research of history or apply forethought?  No simple way I know of.  Core problem is leaving activity maintainers.  Our Code of Conduct has some wording to remind people (such as activity maintainers) not to stop what they do without saying goodbye.  But we can't legislate for laziness.

**Comment by srevinsaju** [comment]
Ohk, can we add a tag to the README.md, so that 
EOL == End of Life, no longer maintained
OLPC == Dedicated repo for OLPC OSes only
and all activities with a proper new readme as supported for sugar3

**Comment by quozl** [comment]
Thanks, but no.  That just adds another level of indirection to the real problem; which is that someone maintaining an activity (such as yourself) then has to trust or not trust the limited information available.  Also, even if we did add that to the README.md file, we can almost guarantee someone will try to port it or change it anyway.  Or blindly add issues insisting it should be done.

I have thought of placing _positive_ statements in README.md files, like "maintained by N", where N is someone's name, but again that can be easily deduced by looking at the commit history, which is something that should be looked at.

I think a focus on GitHub metadata like README.md is misdirected.  We have far larger problems; as our [Maintaining Activities](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#modifying-activities) points out; activities as bundles with no repository, repositories hosted on Gitorious, repositories hosted privately, etc.  And that problem isn't solved by moving everything we can find to GitHub; we tried that.
